content/en/docs/operations/upgrades/_index.md

```
@@ -0,0 +1,37 @@
+
+## Upgrading to v1.0
+
+Version 1.0 brings major change to Cozystack control-plane: it is now completely modular and is composed of independent packages.  
+To manage packages a cozystack-operator was introduced, which replaces previous installer.  
+Assets server is now replaced with a single oci image, and it could be replaced with a git repository if needed.  
+
+However, underlying entities are still helm releases made of helm charts, so during an upgrade no workloads would be recreated or affected in any way.  
+
+### 1. Prerequisites
+Along with version 1.0 new management tools were introduced: `cozyhr` and `cozypkg`. You will need their latest versions to proceed with the upgrade.
+- Take `cozypkg` from the Cozystack Releases page: https://github.com/cozystack/cozystack/releases
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)